sdk/core/Azure.Core/README.md

```diff
@@ -1,45 +1,100 @@
 # Overview 
 
-Azure.Core is a reusable library used to implement .NET Azure SDK components. 
-It contains APIs that make it easy to call Azure service endpoints in a convinient, performant, and consistent manner.
+Azure.Core provides shared primitives, abstractions, and helpers for .NET Azure SDK service client libraries. 
+These shared APIs allow client libraries to expose common functionality in a consistent fashion, 
+so that once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
 
-# Installing
+The main shared concepts of Azure.Core (and so all Azure SDK libraries using Azure.Core) include:
+
+1. Unified APIs for configuring service clients, e.g. configuring retries, logging.
+2. Unified APIs for accessing HTTP responses.
+3. Unified APIs for consuming long running operations (LROs).
+4. Unified APIs for consuming asynchronous streams (```IAsyncEnumerable<T>```) by paging.
+5. Unified exception hierarchy for reporting errors from service requests.
+6. Abstractions for representing Azure SDK credentials.
+
```

Comment:
should we say something about the Test Framework it has too? It is separate to the main scenarios for Azure.Core but heavily used by the client libraries

Reply:
Until we come up with a better pattern to consume than directly including source files, I think it would be wise to avoid doing so.   I know that I feel more strongly about this than others, but I'm very much against that.      (side note:  I've got some ideas to pitch that may help, once I have time to pursue)

Reply:
@maririos, I am not sure I understand. We don't seem to have any test framework APIs in Azure.Core. Or maybe I am just not aware of.

Reply:
I mean things like this: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/core/Azure.Core/tests/TestFramework